build-wasm.sh

```diff
@@ -0,0 +1,52 @@
+# Download WASI-SDK from:
+# https://github.com/WebAssembly/wasi-sdk/releases
+
+if [ $BUILD == 'wasi' ]; then
+	echo "* Building with WASI"
+	IPATH="-I$WASI_PATH/include"
+	STDLIB="$WASI_PATH/lib/wasm32-wasi/libc.a"
+else
+	echo "* Building with a lightweight stdlib"
+	STDLIB="wasm/stdlib.c"
+fi
+
+clang \
+	--target=wasm32 \
+	-Oz \
+	-flto \
+	-nostdlib \
+	-Wl,--no-entry \
+	-Wl,--export-all \
+	-Wl,--lto-O3 \
+	-Wl,-z,stack-size=$[8 * 1024 * 1024] \
+	-o onigmo.wasm \
+	$IPATH -I wasm -I . -I enc/unicode -Wl,-error-limit=0 \
```

Comment:
```suggestion
	$IPATH -I wasm -I . -I enc/unicode -Wl,--error-limit=0 \
```
was getting the below error without this:
```
wasm-ld: error: entry symbol not defined (pass --no-entry to suppress): rror-limit=0
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Thank you @nuclearpidgeon !